texlive.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM texlive/texlive:latest
+
+RUN apt-get update \
+    && apt-get install -y \
+    imagemagick \
+    inkscape \
+    librsvg2-bin \
```

Comment:
We should be able to remove inkscape and librsvg2-bin from the other Dockerfiles as I think these are only used in the pdf creation.